RELEASE-NOTES.txt

```diff
@@ -1,3 +1,6 @@
+16.5
+-----
+ 
```

Comment:
Did you stop adding (or maybe you never had?) the instruction header saying "Use [*]/[**]/[***] for priority markers" as we have at least in Woo?

Asking since I saw that our scripts still haven't been updated to take that new instruction header into account and that so far I've been needing to fix its location manually after running our scripts (and planned to update our scripts one day but haven't taken the time yet)… so wondering if you instead just decided to remove the instruction header at one point, considering the team was now aware enough of those practices?